tests/Unit/ProtoTests/Basic/out/src/Client/BasicClient.php

```diff
@@ -142,7 +144,7 @@ private static function getClientDefaults()
      */
     public function __construct(array $options = [])
     {
-        $clientOptions = $this->buildClientOptions($options);
+        $clientOptions = $this->buildClientOptions($options + $this->getDefaultEmulatorConfig());
```

Comment:
we probably also want to update the documentation of the client object somewhere to include information about the environment variable, e.g. in the class's PHPDoc or the __constructor PHPDoc, something like:

```
Setting the "SPANNER_EMULATOR_HOST" environment variable will automatically set the API Endpoint to
the value specified in the variable, as well as ensure that empty credentials are used in the transport layer.
```

Reply:
Done